build.gradle

```diff
@@ -14,7 +14,6 @@ ext.apacheThriftVersion = '0.9.3'
 ext.jerseyVersion = '2.22.2'
 ext.slf4jVersion = '1.7.16'
 ext.jacksonVersion = '2.7.4'
-ext.checkstyleVersion = '7.1.1'
```

Comment:
I don't think checkstyle is that intelligent. Findbugs is more similar to `go vet`
From wikipedia, 
> The performed checks mainly limit themselves to the presentation and don't analyze content, and do not confirm the correctness or completeness of the program.